libs/headless-lfp/src/test/utils.spec.ts

```diff
@@ -0,0 +1,60 @@
+import { FontFamilyLFF, FontLFF } from '../types';
+import { convertToFontLFP, fetchJSON } from '../utils';
+
+describe('convertToFontLFP', () => {
+  const emptyFontLFF: FontLFF = {
+    apiversion: '',
+    defaultfamily: [],
+    families: {},
+  };
+  const mockId = 'fontsans';
+  const mockName = 'Font Sans';
+  const mockDefault = [mockId];
+  const mockFamily: FontFamilyLFF = {
+    distributable: false,
+    familyid: mockId,
+    family: mockName,
+  };
+  const mockFamilies = { [mockId]: mockFamily };
+
+  it('handles empty `defaultfamilies`', () => {
+    const fontLFP = convertToFontLFP({
+      ...emptyFontLFF,
+      families: mockFamilies,
+    });
+    expect(fontLFP.name).toEqual('');
+  });
+
+  it('handles empty `families`', () => {
+    const fontLFP = convertToFontLFP({
+      ...emptyFontLFF,
+      defaultfamily: mockDefault,
+    });
+    expect(fontLFP.name).toEqual(mockId);
+  });
+
+  it('gets font name', () => {
+    const fontLFP = convertToFontLFP({
+      ...emptyFontLFF,
+      defaultfamily: mockDefault,
+      families: mockFamilies,
+    });
+    expect(fontLFP.name).toEqual(mockName);
+  });
+});
+
+describe('fetchJson', () => {
```

Comment:
nit: Strictly testing the external API isn't a unit test but more like an integration test but I expect it's useful to have this for now.